python/GafferSceneUI/StandardOptionsUI.py

```diff
@@ -96,281 +97,330 @@ def __statisticsSummary( plug ) :
 
 	return ", ".join( info )
 
-Gaffer.Metadata.registerNode(
+plugsMetadata = {
```

Comment:
This is much easier to understand now, thanks. It might be worth a quick comment here along the lines of:
```
# This dict is shared with other nodes that hold related plugs. It is 
# assumed that the first metadata element for each plug is always
# `description`, so take care to preserve this when updating entires here.
```

Reply:
eb0d6dcb98f7300fcde77d24def32010037d5d54